README.md

```diff
@@ -25,4 +25,13 @@ This command will temporarily install [`yo`](https://www.npmjs.com/package/yo) -
 
 ```sh
 npx --ignore-existing -p yo -p generator-web-starter yo web-starter
-```
\ No newline at end of file
+```
+
+## Enhancing this project / Setup instructions
+
+- Clone repository
+- `npm install` - install dependencies
```

Comment:
Prefer `npm ci` - `npm install` will update `package-lock.json`, which you shouldn't be doing unless you're modifying a dependency.